ui/pages/permissions-connect/permissions-connect.container.js

```diff
@@ -68,18 +75,29 @@ const mapStateToProps = (state, ownProps) => {
 
   const connectPath = `${CONNECT_ROUTE}/${permissionsRequestId}`;
   const confirmPermissionPath = `${CONNECT_ROUTE}/${permissionsRequestId}${CONNECT_CONFIRM_PERMISSIONS_ROUTE}`;
+  ///: BEGIN:ONLY_INCLUDE_IN(flask)
+  const snapInstallPath = `${CONNECT_ROUTE}/${permissionsRequestId}${CONNECT_SNAP_INSTALL_ROUTE}`;
+  ///: END:ONLY_INCLUDE_IN
 
   let page = '';
   if (pathname === connectPath) {
     page = '1';
   } else if (pathname === confirmPermissionPath) {
     page = '2';
+    ///: BEGIN:ONLY_INCLUDE_IN(flask)
+  } else if (pathname === snapInstallPath) {
+    page = '3';
```

Comment:
This doesn't seem right. The total number of pages is still hard-coded as 2 (in `permissions-connect.component.js`) for starters, so the header would show "page 3 / 2" here.

I would expect that the total number of pages would be 1 as a baseline, plus 1 if `eth_accounts` was requested, and plus 1 if a snap was requested that hasn't been installed yet. Then calculating the _current_ page depends on that total. This should only be 3 if `eth_accounts` is also being requested, otherwise this would be 2.